Feature/swissign reference story

elessar-ch opened the pull request:
This adds the reference story for SwissSign Group.
Additionally, the customer reference story component has been modified to allow HTML tags in most text and have a general list style available.

mkretz commented on src/customer/swisssigngroup-en.json:
```diff
@@ -0,0 +1,35 @@
+{
+  "title": "bespinian unterstützt SwissSign bei der Migration von SwissID auf Kubernetes",
```
This title needs to be in English

mkretz reviewed (changes requested):
The customer logo needs to be inserted in the customer list with a link to the reference story.

elessar-ch commented:
I updated the PR.
- fixed the headline language
- added SwissSign Group to the customers section on the home page and linked it
- Compressed the SwissSign logo svg with svgo
- Removed a lot of the markup tags in the reference story
- Adjusted the customers and partners logo and text display so that SVGs are displayed correctly and the text below them is aligned
